guessNumber.js

```diff
@@ -0,0 +1,20 @@
+
+const guessNumber = function () {
+    
+    let randomNumber = Math.floor(Math.random()* 10), 
```

Comment:
`randomNumber` and `strNumber` should be `const` because they always has the same value.
And move `randomNumber` outside of function. In this case it has different value on each invoke.